src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -614,6 +614,26 @@ private bool CanChangeFontSize(object obj)
         {
             return true;
         }
+
+        private DelegateCommand toggleIsVisibleGroupCommand;
+
+        /// <summary>
+        /// Command to toggle this group's node preview visibility.
+        /// belongs to.
+        /// </summary>
+        [JsonIgnore]
+        public DelegateCommand ToggleIsVisibleGroupCommand
+        {
+            get
+            {
+                if (toggleIsVisibleGroupCommand == null)
+                    toggleIsVisibleGroupCommand =
+                        new DelegateCommand(ToggleIsVisibleGroup, CanToggleIsVisibleGroup);
+                new DelegateCommand(RemoveGroupFromGroup, CanUngroupGroup);
+
+                return toggleIsVisibleGroupCommand;
```

Comment:
why does this need a backing field at all?

Reply:
to not initiate the delegate command for every call?

Reply:
@zeusongit - why not just use the property?
and what the heck is `new DelegateCommand(RemoveGroupFromGroup, CanUngroupGroup);` line doing?

Reply:
has been removed in the other PR